packages/ETLoad.package/ETLPlaceholderObject.class/class/is.compatibleWith..st

```diff
@@ -0,0 +1,9 @@
+compatibility
+is: squeak3ObjectFormat compatibleWith: squeak5InstSpec
```

Comment:
If possible please rename to something more descriptive ie. `isObjectFormat:compatibleWith:`